packages/renderer/src/lib/ui/EmptyScreen.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+import Fa from 'svelte-fa/src/fa.svelte';
+import { faPaste } from '@fortawesome/free-solid-svg-icons';
+import { onMount, SvelteComponent } from 'svelte';
+
+export let icon: any;
+export let title: string = 'No title';
+export let message: string = 'Message';
+export let commandline: string = '';
+export let hidden: boolean = false;
+
+let fontAwesomeIcon = false;
+let processed = false;
+
+onMount(() => {
+  if (icon?.prefix === 'fas') {
+    fontAwesomeIcon = true;
+  }
+  processed = true;
+});
+
+function copyRunInstructionToClipboard() {
+  const text = copyTextDivElement?.innerText;
+  navigator.clipboard.writeText(text);
+}
+
+let copyTextDivElement: HTMLDivElement;
+</script>
+
+<div class="h-full min-w-full flex flex-col" class:hidden="{hidden}">
+  <div class="pf-c-empty-state h-full">
+    <div class="pf-c-empty-state__content">
+      <p class="pf-c-empty-state__body">
+        {#if processed}
+          {#if fontAwesomeIcon}
+            <Fa
+              icon="{icon}"
+              size="55"
+              class="cursor-pointer pf-c-empty-state__icon"
+              style="display:inline-block; text-align: center" />
+          {:else}
+            <svelte:component
+              this="{icon}"
+              size="60"
```

Comment:
size here is 60 and for font awesome it's 55. Is it expected ?

Reply:
For now, yes. If you specify 55 for FA the height is set to 1em for a 55px font-size, which for faLayerGroup results in an svg that was ~62x55. Our current SVGs all have embedded borders, so even at 60x60px they are just marginally smaller.

Ideally we should remove the embedded borders in our SVGs and change sizing to work like FA, but that would require changing the SVGs and nav page at the same time, so deferring that for later.